src/components/layout/FixedLeft/FixedLeft.jsx

```diff
@@ -85,8 +98,9 @@ FixedLeft.propTypes = {
   stickAt: PropTypes.number,
   contentStyle: PropTypes.object,
   fixedStyle: PropTypes.object,
+  data: PropTypes.object,
 };
 
-FixedLeft.displayName = '!FixedLeft';
+FixedLeft.displayName = '!Plasma@FixedLeft';
```

Comment:
It is syntax that tells the auto-generating doc parser that this it shouldn't render a preview for this component.